### PR TITLE
Updates from the 20240410 meeting to the spec

### PR DESCRIPTION
**Pull request by swcurran**
- **Update Readme**
- **Remove spec-up from Readme**
- **Typo**
- **Typo**
- **Updates from the 20240410 meeting to the spec**

### CODE REVIEW

**Comment by swcurran** [comment]
Dang -- @brianorwhatever -- I might have screwed up on this PR. I forgot to create a branch for my "README" update, and then created a branch from that updated "main".  Sigh. 

I there is a problem once we get these merged, I'll figure out how to fix things...

File: spec/overview.md [hunk 1]
```diff
@@ -115,7 +115,7 @@ the [did:tdw Examples](#didtdw-example) of this specification.
...
+We propose that the `/whois` path for a DID enable a comparable, decentralized, version of the `WHOIS` protocol for DIDs. Notably, when `<did>/whois` is resolved (using a standard DID `service` that follows the [[ref: Linked-VP]] specification), a [[ref: Verifiable Presentation]] (VP) may be returned (if published by the DID Controller) containing [[ref: Verifiable Credentials]] with the DID as the `credentialSubject`, and the VP signed by the DID. Given a DID, one can find out who controls that DID by calling `<did>/whois` and processing the returned VP. That's powerful -- a decentralized trust registry. For `did:tdw`, the approach is very simple -- transform the DID to its HTTPS equivalent, and `GET <https>/whois.json`. Need to know who issued the VCs in the VP? Get the issuer DIDs from the VCs, and resolve `<issuer did>/whois`. This is comparable to walking a CA (Certificate Authority) hierarchy, but self-managed by the DID Controllers -- and the issuers that attest to them.
```
Comment (brianorwhatever) [inline spec/overview.md:118]: > by calling `<did>/whois`

what does this mean?

Comment (brianorwhatever) [reply spec/overview.md:118]: replace calling with resolving maybe?

File: spec/specification.md [hunk 2]
```diff
@@ -301,11 +303,15 @@ DID](#publishing-a-parallel-didweb-did) section of this specification.
...
 
-To deactivate the DID, the [[ref: DID Controller]] **SHOULD** update the DIDDoc
-to include the item `"deactivated": true`, as defined in the [[spec:DID-CORE]]
-specification. When doing that, the [[ref: DID Controller]] might also remove
-all keys from the DIDDoc, and set the top level `controller` item to `""` to
-prevent further updates to the DID.
+To deactivate the DID, the [[ref: DID Controller]] **SHOULD** add to the [[ref:
+DID log entry]] [[ref: parameters]] the item `"deactivated": true`. A [[ref: DID
+Controller]] **MAY** update the DIDDoc further to indicate the deactivation of
+the DID, such as removing the `authorization` key type entries, preventing
```
Comment (brianorwhatever) [inline spec/specification.md:309]:
```suggestion
the DID, such as removing the `authentication` key type entries, preventing
```

File: spec/specification.md [hunk 3]
```diff
@@ -350,6 +356,10 @@ items are defined below.
   - By default, the value is initialized to `sha256`.
   - Acceptable values:
     - `sha256`: Use the `SHA-256` algorithm from [[spec:rfc4634]].
+- `cryptosuite`: The Data Integrity cryptosuite to use when generating and verifying the authorization proofs on the [[ref: DID log entries]].
```
Comment (brianorwhatever) [inline spec/specification.md:359]:
```suggestion
- `cryptosuite`: The Data Integrity cryptosuite to use when generating and verifying the authentication proofs on the [[ref: DID log entries]].
```

File: spec/specification.md [hunk 4]
```diff
@@ -350,6 +356,10 @@ items are defined below.
...
+  - By default, the value is initialized to `eddsa-2022`
+  - Acceptable values:
+    - `eddsa-2022`: Use the [EDDSA 2022](https://github.com/digitalbazaar/eddsa-2022-cryptosuite) cryptosuite.
```
Comment (brianorwhatever) [inline spec/specification.md:362]:
```suggestion
    - `eddsa-jcs-2022`: Use the [eddsa-jcs-2022](https://www.w3.org/TR/vc-di-eddsa/#eddsa-jcs-2022) cryptosuite.
```

my implementation has this wrong it's on my todo list to use the above
https://github.com/bcgov/trustdidweb-ts/issues/4

Comment (brianorwhatever) [reply spec/specification.md:362]: I know, sorry, the examples are wrong but I'm sure we knew they needed updating

File: spec/specification.md [hunk 5]
```diff
@@ -369,6 +379,22 @@ items are defined below.
...
+  be deactivated. See the [deactivate (revoke)](#deactivate-revoke) section of
+  this specification for more details.
+- `ttl`: A number, the number of seconds that a cache entry for a resolved
+  `did:tdw` DID should last, as recommended by the [[ref: DID Controller]]. A
+  resolver can use this value in deciding whether to retrieve a new version of
+  the DID's `did.jsonl` file. If not specified, resolvers may set a default
+  based on the business needs of the resolver clients.
+  - Caching of a `did:tdw` can be valuable in places where the business rules
+    require resolving a number of DID URLs for the same DID. For example, a
+    client might want call the resolver to the current DIDDoc, and then make
```
Comment (brianorwhatever) [inline spec/specification.md:392]:
```suggestion
    client might want to call the resolver for the current DIDDoc, and then make
```

File: spec/specification.md [hunk 6]
```diff
@@ -369,6 +379,22 @@ items are defined below.
...
+    repeated calls to get all of the previous versions of the DIDDoc. By caching
+    the DIDDoc state, the resolver would not have to retrieve and process the
+    [[ref: DID Log]] on every call.
+  - A Web Server handling one or more `did.jsonl` file **MAY** be configured to
```
Comment (brianorwhatever) [inline spec/specification.md:396]:
```suggestion
  - A Web Server handling one or more `did.jsonl` files **MAY** be configured to
```

**Review by brianorwhatever — APPROVED** [review]
long live `.jsonl` 👍

here is the discussion about it being standardized. Internet draft is basically non-existent so there won't be a proper mime type for a long time. Interesting that amazon is using it for dynamodb exports
https://github.com/wardi/jsonlines/issues/19

I wonder what my browser will say about this 👀

**Review by brianorwhatever — APPROVED** [review]

**Comment by swcurran** [comment]
Merging to get the content out there, but will update based on your comments and put in another PR.

**Comment by swcurran** [comment]
> long live `.jsonl` 👍
> 
> here is the discussion about it being standardized. Internet draft is basically non-existent so there won't be a proper mime type for a long time. Interesting that amazon is using it for dynamodb exports [wardi/jsonlines#19](https://github.com/wardi/jsonlines/issues/19)
> 
> I wonder what my browser will say about this 👀

IETF Draft: rfc7464 woot!!

File: spec/overview.md (hunk 1 above)
Comment (swcurran) [reply spec/overview.md]: Cleaned up the entire paragraph in the next PR.

File: spec/specification.md (hunk 2 above)
Comment (swcurran) [reply spec/specification.md:309]: Fixed in next PR.

File: spec/specification.md (hunk 3 above)
Comment (swcurran) [reply spec/specification.md:359]: FIxed in next PR -- thanks!!

File: spec/specification.md (hunk 6 above)
Comment (swcurran) [reply spec/specification.md:396]: Fixed in next PR

**Comment by andrewwhitehead** [comment]
That RFC doesn't quite seem to apply, it requires a 'RS' character (record separator, ASCII 0x1E) at the beginning of each line. `jq`'s `--seq` option also seems to require that. :(

**Pull request merged**